src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs

```diff
@@ -375,6 +383,214 @@ public Range ExposureRange
         }
         #endregion Exposure
 
+        #region saturation
+        /// <summary>
+        /// Gets or sets the camera saturation value.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        /// <exception cref="ObjectDisposedException">The camera has already been disposed. </exception>
+        public int Saturation
+        {
+            get
+            {
+                Native.GetSaturation(_camera.GetHandle(), out int val).
+                    ThrowIfFailed("Failed to get camera saturation value");
+
+                return val;
+            }
+            set
+            {
+                Native.SetSaturation(_camera.GetHandle(), value).
+                    ThrowIfFailed("Failed to set camera saturation value.");
+            }
+        }
+
+        /// <summary>
+        /// Gets the available saturation value.
```

Comment:
that follows the pattern of already existing properties, but "Gets the available saturation value range." or "Gets the available saturation range." or "Gets the range of allowed saturation value."  would be better.

Reply:
I'll fix it.